src/platform/BL_Python/platform/feature_flag/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .caching_feature_flag_router import CachingFeatureFlagRouter
+from .db_feature_flag_router import DBFeatureFlagRouter
+from .feature_flag_router import FeatureFlagRouter
+
+__all__ = ("FeatureFlagRouter", "CachingFeatureFlagRouter", "DBFeatureFlagRouter")
```

Comment:
As far as I understand it, functionally the `import *` statement is the primary use for it. However, it also provides:

* In-code documentation of the "public" interface
* Additional information for tooling (documentation generation, type checking, etc)
* In rare cases, avoidance of name clashes
* In type stubs (pyi files) it can simplify describing APIs

It is something I use when I remember to, but I have been inconsistent. It would be good to be consistent about its usage for any public APIs in a philosophical sense ... as for whether I've actually gotten use from it, that's debatable.